### DISCUSSION TITLE
gx command open url not work

### DISCUSSION THREAD

**Discussion by OriLiMu** [LazyVim/LazyVim#1916] [Q&A] [answered]
I use [vim-markdown](https://github.com/preservim/vim-markdown#mappings) to edit markdown file.

Which have a key mapping `gx` to open url under cursor. But not work for me with the error message `cursor not on the link` . How to fix it?

**Comment by dpetka2001** [comment, marked as answer]
There seem to be a few open issues in the repo regarding this and also a PR awaiting to get merged. Better to ask a question over there, since this doesn't seem to be a Lazyvim problem.

**Comment by AndreM222** [comment]
This is most probably because if you check the disable plugins list you might have netrwPlugin in there. If you disable that then it disables gx to open url links with gx

**Comment by folke** [comment]
You can actually use `noice.nvim` for making markdown urls clickable:

```lua
    vim.api.nvim_create_autocmd("FileType", {
      pattern = "markdown",
      callback = function(event)
        vim.schedule(function()
          require("noice.text.markdown").keys(event.buf)
        end)
      end,
    })
```

**Reply by AndreM222** [reply]
Brother you are a god. 🙏
